float32 x
float32 y
float32 heading
float32 x_filter
float32 y_filter
float32 heading_filter
bool homing_updated
